crypto/evp/evp_enc.c

```diff
@@ -1030,13 +1030,27 @@ int EVP_CIPHER_CTX_set_key_length(EVP_CIPHER_CTX *c, int keylen)
     OSSL_PARAM params[2] = { OSSL_PARAM_END, OSSL_PARAM_END };
```

Comment:
Move this line to be inside the non-legacy block.

Reply:
Done